recipes/astropy/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "1.3" %}
+
+package:
+  name: astropy
+  version: {{version}}
+
+source:
+  fn: astropy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/a/astropy/astropy-{{version}}.tar.gz
+  md5: bf4c8a033a7085fbc9ec604eebbdbdc5
+
+build:
+  detect_binary_files_with_prefix: False
+  entry_points:
+    - fits2bitmap = astropy.visualization.scripts.fits2bitmap:main
+    - fitscheck = astropy.io.fits.scripts.fitscheck:main
+    - fitsdiff = astropy.io.fits.scripts.fitsdiff:main
+    - fitsheader = astropy.io.fits.scripts.fitsheader:main
+    - fitsinfo = astropy.io.fits.scripts.fitsinfo:main
+    - samp_hub = astropy.vo.samp.hub_script:hub_script
+    - volint = astropy.io.votable.volint:main
+    - wcslint = astropy.wcs.wcslint:main
+  number: 0
+
+requirements:
+
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+
+  run:
+    - python
+    - numpy x.x
+
+test:
+  commands:
+    - fits2bitmap --help
+    - fitscheck --help
+    - fitsdiff --help
+    - fitsheader --help
+    - fitsinfo --help
+    - samp_hub --help
+    - volint --help
+    - wcslint --help
+  imports:
+    - astropy
+
+about:
+  home: http://www.astropy.org/
+  license: BSD
```

Comment:
As this is merged already, these are some thoughts for the feedstock.

Would be good to specify the number of clauses in the feedstock (e.g. BSD 3-Clause). Could also have a `license_family` field that would be basically like this one. Finally packaging the license in the conda package has become a good practice as many licenses have a clause requiring it. Also it can be handy for the end user to reference. This can be done by adding `license_file: licenses/LICENSE.rst` to pick it up from the source.